scripts/lib/aws.sh

```diff
@@ -50,7 +50,7 @@ aws_generate_temp_creds() {
     JSON=$(daws sts assume-role \
            --role-arn "$ACK_ROLE_ARN"  \
            --role-session-name tmp-role-"$__uuid" \
-           --duration-seconds 3600 \
+           --duration-seconds 7200 \
```

Comment:
I'd like to try this out in a Prow environment before merging. I worry that IAM role chaining, this might be denied by the STS service.

Reply:
Cool, let us know the outcome

Reply:
As I suspected, this violates the 1 hour IAM chaining rule. 
```
An error occurred (ValidationError) when calling the AssumeRole operation: The requested DurationSeconds exceeds the 1 hour session limit for roles assumed by role chaining.
```

I think that an hour is more than enough time to achieve an e2e test result, though. So instead we should look toward parallelising the tests.